usaspending_api/transactions/agnostic_transaction_loader.py

```diff
@@ -156,8 +156,10 @@ def generate_ids_from_broker(self):
         with psycopg2.connect(dsn=get_broker_dsn_string()) as connection:
             with connection.cursor("usaspending_data_transfer") as cursor:
                 cursor.execute(sql.strip("\n"))
+                # fetchmany returns a list of tuples that can be iterated via next()
                 while True:
-                    id_list = [id[0] for id in cursor.fetchmany(size=self.chunk_size)]
+                    tuples = cursor.fetchmany(size=self.chunk_size)
+                    id_list = [id[0] for id in tuples]
```

Comment:
Any reason for this change? Looks to be the same other than the results from the `fetchmany()` call being stored in a temporary variable, `tuples`

Reply:
Oh i did that when I was debugging but I can revert this change no problem if you'd like.